src/utilities/permissions.ts

```diff
@@ -1583,6 +1622,19 @@ const featurePermissions: FeaturePermissions = {
     canRead: () => false, // Not implemented
     canUpdate: () => false, // Not implemented
   },
+  sequenceFilter: {
+    canCreate: user => queryPermissions.CREATE_SEQUENCE_FILTER(user),
+    canDelete: user => queryPermissions.DELETE_SEQUENCE_FILTERS(user),
+    canRead: () => true,
+    canUpdate: (user, model) => queryPermissions.UPDATE_SEQUENCE_FILTER(user, model),
+  },
+  sequenceTemplate: {
+    canCreate: user => queryPermissions.CREATE_SEQUENCE_TEMPLATE(user),
+    canDelete: (user, sequenceTemplate) => queryPermissions.DELETE_SEQUENCE_TEMPLATE(user, sequenceTemplate),
+    canRead: () => true,
+    canTemplate: (user, plan, model) => queryPermissions.EXPAND_TEMPLATES(user, plan, model),
```

Comment:
Can this also be renamed to be a little more clear as to what the functionality is that is slightly more user facing? Maybe `canExpandTemplate` if "expand" is what the user sees.